SentinelOne/sentinelone/ingest/parser.yml

```
@@ -843,16 +843,16 @@ stages:
           # Fix: sentinelone.threatInfo.identifiedAt: "{{json_event.message.threatInfo.identifiedAt}}"
           threat.enrichments.matched.occurred: "{{json_event.message.threatInfo.identifiedAt}}"
           sentinelone.threatInfo.NewTargetUserName: "{{json_event.message.threatInfo.NewTargetUserName}}"
-          # Fix: sentinelone.threatInfo.md5: "{{json_event.message.threatInfo.md5}}"
-          # not sure if this is about a file
-          threat.enrichments.indicator.file.hash.md5: "{{json_event.message.threatInfo.md5}}"
           # Fix: sentinelone.threatInfo.threatName: "{{json_event.message.threatInfo.threatName}}"
           # TODO: use threat.enrichments.indicator.modified_at when supported
 
           file.name: "{{ json_event.message.threatInfo.threatName }}"
           file.extension: "{{json_event.message.threatInfo.fileExtension |lower}}"
           file.path: "{{json_event.message.threatInfo.filePath}}"
           file.size: "{{json_event.message.threatInfo.fileSize}}"
+          file.hash.md5: "{{json_event.message.threatInfo.md5}}"
+          file.hash.sha1: "{{json_event.message.threatInfo.sha1}}"
+          file.hash.sha256: "{{json_event.message.threatInfo.sha256}}"
```

I'm not sure about the destination. If we respect the semantic of fields, these data should land in `threat.enrichments.indicator.file.hash`
```suggestion
          threat.enrichments.indicator.file.hash.md5: "{{json_event.message.threatInfo.md5}}"
          threat.enrichments.indicator.file.hash.sha1: "{{json_event.message.threatInfo.sha1}}"
          threat.enrichments.indicator.file.hash.sha256: "{{json_event.message.threatInfo.sha256}}"
```

---

FYI in Harfang Lab we use this `file.hash.X` field -> https://github.com/SEKOIA-IO/intake-formats/blob/main/HarfangLab/harfanglab/ingest/parser.yml#L95
This is also consistent with the use of other `file.X` fields copied from `json_event.message.threatInfo.X` fields (such as `file.name` copied from `json_event.message.threatInfo.threatName`)

Also not sure threat.enrichments.indicator.file.hash is displayed in Alerts graphs

---

I didn't see that other `json_event.message.threatInfo.X` landed in `file.X`.

About Harfanglab, I checked with them that the hashes represents the hashes of the executable (and not the program as it was supposed previously).